sentry-android-core/src/main/java/io/sentry/android/core/InternalSentrySdk.java

```diff
@@ -44,6 +44,7 @@ public final class InternalSentrySdk {
   @Nullable
   public static IScope getCurrentScope() {
     final @NotNull AtomicReference<IScope> scopeRef = new AtomicReference<>();
+    // TODO [HSM] should this retrieve combined scope?
```

Comment:
This way, if a user adds, e.g. tags to global scope, cross platform SDKs will also receive the tag.